components/Feature/TopicExplorer/index.js

```diff
@@ -1,25 +1,39 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import Markdown from 'markdown-to-jsx';
 
 const TopicExplorer = (props) => {
   const [searchTerm, setSearchTerm] = useState('')
   const [searchResults, setSearchResults] = useState([])
+  const [tags, setTag] = useState([])
 
   const getSearchResults = (searchTerm) => {
     var results = []
     const newSearchTerm = searchTerm.toLocaleLowerCase();
-
     for(const topic of props.topics) {
       const promptTags = topic.promptTags
       promptTags.forEach(tag => {
-        if(tag.toLocaleLowerCase() == newSearchTerm){
-          results.push(topic);
-        }
+          if(tag.toLocaleLowerCase() == newSearchTerm){
+            results.push(topic);
+          }
       });
     }
     return results
   }
 
+  useEffect(() => {
+      if(props.topics){
```

Comment:
I think a guard clause here is clearer, though I'm not sure how to write them in javascript idiomatically. Maybe:
```javascript
if (!props.topics) { return }
```

Reply:
I've updated this as part of the previous commit 😄